### ISSUE TITLE
Nginx (error 502)

### ISSUE THREAD

**Issue by raphgilles** [YunoHost-Apps/pixelfed_ynh#37]
Hello,

For the first time, with your last upgrade, the installation arrives at the end. But now I have an error 502.

On the frontpage of the package you writed "I assume only the nginx file needs improvments".
Ok. But what can I do to repair that 😅

Thank you for your work !

**Comment by raphgilles** [comment]
So, I activated php5.2 on my server, and I was able to create my admin account.

But now, when I'm logged off, I have this error. I tried to install the Laravel suite because I have the "No preview available" issue too.
So I needed Laravel Horizon (cf. dansup https://mastodon.social/@pixelfed/101865339153744128) and Laravel Mix for the other issue.

But it changes nothing. Always the "No preview available" and the Mix issue.

Any idea ?

![pixelbug](https://user-images.githubusercontent.com/49836486/56476487-cab24c80-6498-11e9-9aac-7b9a0bcbdb51.jpg)

![pixelbug2](https://user-images.githubusercontent.com/49836486/56476512-5cba5500-6499-11e9-862e-7ef047f01703.jpg)
